shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -117,18 +117,7 @@ object JsonLdUtils {
         }
 
         // TODO find a way to avoid this extra parsing
```

Comment:
I think the comment is deprecated now.

Reply:
no. the code is extracted in a separate method, but the extra parsing is still a thing.